data-syncs/supported-data-sync-sources/rest-api.md

```diff
@@ -81,22 +90,121 @@ You can learn more about these sections in [Appendix A - Other Sections.](rest-a
 
 <figure><img src="../../.gitbook/assets/image (298).png" alt=""><figcaption><p>Image 2: The Source Tab</p></figcaption></figure>
 
+## Best practices
+
+### Retrieve nested fields
+
+To get fields in a nested array, you can either set the nested array as the root, or you can use Path to Iterate to expand the array. 
+
+#### Examples
```

Comment:
```json
{
  "groupId": 111,
  "users": [
{
      "userId": 1,
      "name": "Jack"
  },
 {
      "userId": 2,
      "name": "Jill"
  }
]
```

#### Using $ as Root JSONPath

**Records Root JSONPath**: `$.users`
**Schema**:

- `$.userId` for **ID**
- `$.name` for **Name**

Note that you cannot reference "groupId" because it is above the specified root scope.